### DISCUSSION TITLE
【要望】質問の回答への返信にいいねを押せるようにしてほしい

### DISCUSSION THREAD

**Discussion by Yotty0404** [increments/qiita-discussions#613] [要望・アイデア・不具合]
以前、他の方への回答へ返信できるようになりましたが、そのおかげで別の方が返信として補足説明など追加情報を提示してくることがあります。
しかし、いいねは回答元へしか押せないため、別の方の補足説明などにいいねを押すことができません。

せっかく返信で有益な情報をいただけたので、ほかの方への返信にもいいねを押せるようになるとありがたいです。

ご検討いただけますと幸いです

**Comment by tanaka-asami415** [comment]
@Yotty0404 
ご不便をおかけしており申し訳ないです。
同様のご意見もいただいていて、個人的にも押したいことがあります。
質問のコメントにも「いいね」をできるようにしたいと思っているのですが、直近の開発優先度などから着手ができていない状況です。
少し先になってしまうかもしれませんが、お待ちいただけると幸いです🙇
